docs/Aanleiding-en-onderzoeksvraag.md

```diff
@@ -0,0 +1,56 @@
+
+Maar wat geschikt is voor een financiële deskundige, werkt misschien niet voor een onderzoeker die een juridische check wil doen. Ook is het voor een uitvoeringsorganisatie vervelend als verschillende disciplines en organisaties zich op verschillende momenten over de wijze van ontwikkelen en het gebruik van de algoritmen buigen en de uitvoering belasten met vragen. Zowel voor onderzoekers, als voor uitvoeringsorganisaties lijkt een meer integrale aanpak noodzakelijk. Bijkomend voordeel is dat hierdoor voor uitvoeringsorganisaties duidelijker wordt welke beslissingen en keuzen tijdens de ontwikkeling gedocumenteerd moeten worden op een manier die een review mogelijk maakt.
+
+Om te komen tot zo’n werkmethode is niet eenvoudig. Niet alle organisaties hebben een set algoritmen/beslisregels in natuurlijke taal voorhanden maar alleen de beschikking over computercode die nauwelijks leesbaar is voor leken. Het komt ook voor dat organisaties software (of het gebruik van software) hebben aangeschaft waarmee wetten worden uitgevoerd en waarbij de leverancier als enige de inhoudelijke kennis in huis heeft. Maar, er zijn ook uitvoeringsinstellingen die de beslisregels in natuurlijke taal in de la hebben liggen. Dit gegeven, of belofte (?) heeft geleid tot het
+idee om deze documenten te gaan onderzoeken.
+
+[1](#sdfootnote1anc) [Datagedreven selectie van aangiften door de Belastingdienst | Rapport | Algemene Rekenkamer](https://www.rekenkamer.nl/publicaties/rapporten/2019/06/11/datagedreven-selectie-van-aangiften-door-de-belastingdienst)
+
+[2](#sdfootnote2anc) [Wie controleert de hogepriesters van de AI algoritmen? - Smarthealth](https://www.smarthealth.nl/2018/06/04/wie-controleert-ai-artificial-intelligence-zorg/)
+
+[3](#sdfootnote3anc) [Wat is het toetsingskader | Algoritmes | Algemene Rekenkamer](https://www.rekenkamer.nl/onderwerpen/algoritmes/algoritmes-toetsingskader)
+
+[4](#sdfootnote4anc) [Publicatie Raad van State over digitalisering in wetgeving en bestuursrechtspraak - Raad van State](https://www.raadvanstate.nl/@125918/publicatie-digitalisering/)
+
+## Aanpak en uitgangspunten
+
+### *Aanpak*
+
+Om ergens te kunnen starten willen wij aan de hand van deze beslisregels of andere vormen van beschrijvingen met medewerkers van uitvoeringsinstellingen ontdekken wat een geschikte onderzoeksmethode zou zijn. Hierdoor wordt geborgd dat het resultaat (het opleveren van een multidisciplinaire werkmethode) aansluit bij de praktijk. Bijkomend voordeel is er ook voor de uitvoeringsinstellingen. Door dit onderzoek te faciliteren, wordt voorkomen dat er later vanuit een hiërarchische positie eisen aan de uitvoering gesteld worden die de uitvoering eerder hinderen dan faciliteren. Ook kan het leiden tot een pool van deskundigen die elkaars processen beoordelen (denk aan een gateway review). Dit betekent dat het lerend vermogen van de overheid toe kan nemen.
+
+![method](../docs/figures/method.webp)
```

Comment:
```suggestion
![method](./docs/figures/method.webp)
```
Ik zat te knutselen in een lokale webserver. Voor GitHub Pages zou dit moeten werken.
[Voorbeeld in tijdelijke fork](https://sanderke.github.io/respec/#aanpak)